### PR TITLE
Convert StoryPromo to TS

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️ 

No functional changes.

### CODE REVIEW

File: common/model/editorial-series.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-type EditorialSeriesColour = 'turquoise' | 'red' | 'green' | 'purple';
+export type EditorialSeriesColour = 'turquoise' | 'red' | 'green' | 'purple';
```
Comment (jamesgorrie) [inline common/model/editorial-series.js:2]: This file should also be `ts`?

Comment (davidpmccormick) [reply common/model/editorial-series.js:2]: ah yes, thanks.

**Review by jamesgorrie — APPROVED** [review]
@davidpmccormick updated the js file - should be good to go!

**Comment by davidpmccormick** [comment]
> updated the js file - should be good to go!

Weird – I'm sure I did that yesterday. Possibly got caught by the pre-commit hook failing but didn't check. Hey ho – thanks for doing it for me!

**Pull request merged**